README.md

```diff
@@ -87,6 +87,17 @@ Below is an example configuration for a Lambda function that is part of a step f
 ## Logging Programmatic Errors
 With the tracer configured, simply call `console.log("[LUMIGO_LOG] <YOUR_MESSAGE>");` to cerate custom errors that are visible throughout the platform. This can be used anywhere in your Lambda code, and is included with the `@lumigo/tracer` package.
 
+## Logging detailed alerts
```

Comment:
Look like `type, extra and err` are part of the second parameter, and not parameter by it own